script.js

```diff
@@ -1,4 +1,178 @@
+
+  // If player username has been input but version is empty and game has not started, request user to input normal, korean, or computer.
+  if (inputMode == "username") {
+    // if input is valid
+    if (input == "normal" || input == "korean" || input == "computer") {
+      versionInput = input;
+      inputMode = "game";
+      myOutputValue =
+        "Hello " +
+        playerUsername +
+        ". We are playing " +
+        versionInput +
+        " mode. Please input rock, paper , or scissors.";
+    } else {
+      myOutputValue =
+        "Hello " +
+        playerUsername +
+        ". Please input 'normal', 'korean' , or 'computer'.";
+    }
+  }
+
+  // if it is the first input, playTotalCount will be zero and username is zero. Take the first input as username.
+  if (inputMode == 0 && !(input == 0)) {
+    playerUsername = input;
+    inputMode = "username";
+    myOutputValue =
+      "Hello " +
+      playerUsername +
+      ". Please input 'normal', 'korean' , or 'computer'.";
+  }
+
+  // To modify the input if game is under computer mode
+  if (versionInput == "computer") {
+    input = rockPaperScissors();
+  }
+
+  // If it is a draw.
+  if (
+    input == computerInput ||
+    (input == "reversed rock" && computerInput == "rock") ||
+    (input == "reversed scissors" && computerInput == "scissors") ||
+    (input == "reversed paper" && computerInput == "paper")
+  ) {
+    playerDrawCounter = playerDrawCounter + 1;
+    playerTotalCounter =
+      playerWinCounter + playerLossCounter + playerDrawCounter;
+    myOutputValue =
+      "Hello " +
+      playerUsername +
+      "! The computer chose " +
+      computerInput +
+      ".<br>" +
+      "You chose " +
+      input +
+      ".<br><br>" +
+      "You draw! Wow!" +
+      "<br><br>" +
+      "So far you have been winning " +
+      playerWinCounter +
+      "/" +
+      playerTotalCounter +
+      ". Pretty alright!";
+    koreanTieBreaker = recentWinner;
+  }
+
+  // Conditions for player to lose.
+  if (
+    (input == "rock" && computerInput == "paper") ||
+    (input == "scissors" && computerInput == "rock") ||
+    (input == "paper" && computerInput == "scissors") ||
+    (input == "reversed rock" && computerInput == "scissors") ||
+    (input == "reversed scissors" && computerInput == "paper") ||
+    (input == "reversed paper" && computerInput == "rock") ||
+    (versionInput == "korean" && koreanTieBreaker == "computer")
+  ) {
+    playerLossCounter = playerLossCounter + 1;
+    playerTotalCounter =
+      playerWinCounter + playerLossCounter + playerDrawCounter;
+    myOutputValue =
+      "Hello " +
+      playerUsername +
+      "! The computer chose " +
+      computerInput +
+      ".<br>" +
+      "You chose " +
+      input +
+      ".<br><br>" +
+      "You lose! Bummer." +
+      "<br><br>" +
+      "So far you have been winning " +
+      playerWinCounter +
+      "/" +
+      playerTotalCounter +
+      ". Pretty alright!";
```

Comment:
You might find it easier to use backtick (`) instead of having to use "" for strings and  then having to add variables outside the string. For example:

"Hello " +
      playerUsername +
      "! The computer chose " +
      computerInput +
      ".<br>" +
      "You chose " +
      input +
      ".<br><br>"

vs

(backtick here) Hello ${playerUsername}! The computer chose ${computerInput}. (insert br here) You chose ${input}. (insert two brs here (backtick here)

If I use backtick in this comment it indents my line as code and removes the backtick haha